src/components/Home.js

```diff
@@ -0,0 +1,23 @@
+function Home() {
+  return (
+    <>
+      <div>
+        <h1>Welcome to our page!</h1>
+        <p>
+          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has
+          been the industry&aposs standard dummy text ever since the 1500s,
+          when an unknown printer took a galley of type and scrambled it
+          to make a type specimen book. It has survived not only five
+        </p>
+        <p>
+          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has
+          been the industry&aposs standard dummy text ever since the 1500s,
+          when an unknown printer took a galley of type and scrambled it
+          to make a type specimen book. It has survived not only five
+        </p>
```

Comment:
Kindly add the right spacings on your texts to match the layout of the wireframe as required😉.

yours
<img width="960" alt="image" src="https://github.com/Katended/math-magicians/assets/64643995/378f08bc-96ed-42f0-b939-861262db2e31">

design
<img width="347" alt="image" src="https://github.com/Katended/math-magicians/assets/64643995/69c1b4e1-2ac0-4b07-bb1f-4734db3421d6">